model/jpa/src/main/resources/META-INF/jpa-changelog-3.2.0.xml

```diff
@@ -0,0 +1,235 @@
+        </createIndex>
+        <createIndex indexName="IDX_CLIENT_SESSION_SESSION" tableName="CLIENT_SESSION">
+            <column name="SESSION_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CLI_SES_AUT_STAT_CLI_SES" tableName="CLIENT_SESSION_AUTH_STATUS">
+            <column name="CLIENT_SESSION" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CLI_SESS_NOTE_CLI_SESS" tableName="CLIENT_SESSION_NOTE">
+            <column name="CLIENT_SESSION" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CLI_SES_PRT_MAP_CLI_SES" tableName="CLIENT_SESSION_PROT_MAPPER">
+            <column name="CLIENT_SESSION" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CLI_SESS_ROLE_CLI_SESS" tableName="CLIENT_SESSION_ROLE">
+            <column name="CLIENT_SESSION" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CLIENT_TEMPLATE_REALM" tableName="CLIENT_TEMPLATE">
+            <column name="REALM_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CLIENT_TEMPL_ATTR_TEMPL" tableName="CLIENT_TEMPLATE_ATTRIBUTES">
+            <column name="TEMPLATE_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CL_USR_SESS_NOT_CL_SESS" tableName="CLIENT_USER_SESSION_NOTE">
+            <column name="CLIENT_SESSION" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_COMPONENT_REALM" tableName="COMPONENT">
+            <column name="REALM_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_COMPO_CONFIG_COMPO" tableName="COMPONENT_CONFIG">
+            <column name="COMPONENT_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_COMPOSITE" tableName="COMPOSITE_ROLE">
+            <column name="COMPOSITE" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_COMPOSITE_CHILD" tableName="COMPOSITE_ROLE">
+            <column name="CHILD_ROLE" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CREDENTIAL_ATTR_CRED" tableName="CREDENTIAL_ATTRIBUTE">
+            <column name="CREDENTIAL_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_FED_CRED_ATTR_CRED" tableName="FED_CREDENTIAL_ATTRIBUTE">
+            <column name="CREDENTIAL_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_GROUP_ATTR_GROUP" tableName="GROUP_ATTRIBUTE">
+            <column name="GROUP_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_GROUP_ROLE_MAPP_GROUP" tableName="GROUP_ROLE_MAPPING">
+            <column name="GROUP_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_GROUP_ROLE_MAPP_ROLE" tableName="GROUP_ROLE_MAPPING">
+            <column name="ROLE_ID" type="VARCHAR(36)"/>
```

Comment:
This index is unnecessary because there is already an index for primary key on GROUP_ROLE_MAPPING(**ROLE_ID**, GROUP_ID) named _constraint_group_role_. Please remove.